.gitignore

```diff
@@ -32,6 +32,7 @@ saturn-ui.iml
 
 /integration-tests/failures
 /integration-tests/screenshots
```

Comment:
Is `screenshots` still necessary now that it's under `results`? Also is `failures` still needed?

Reply:
Oh i suppose not! Unless there are screenshots still lingering around. I dont know when /failures is written to tho, I don't think it's part of this script. Let me just remove the screenshots at least, and we'll go from there? :)